ProjectLighthouse/Types/Serialization/LbpCustomXml.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Xml.Serialization;
+
+namespace LBPUnion.ProjectLighthouse.Types.Serialization;
+
+public class LbpCustomXml : ILbpSerializable
+{
+    public string Content { get; set; } = "";
```

Comment:
```suggestion
    public required string Content { get; set; }
```
I think instead of making this initialized to an empty string we could just mark it as required so that people don't return this class without initializing content to something.